charts/opentelemetry-demo/Chart.yaml

```diff
@@ -11,4 +11,4 @@ maintainers:
   - name: tylerhelmuth
   - name: wph95
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: "0.4.0-alpha"
+appVersion: "0.5.0-alpha"
```

Comment:
Version bump should be done in its own PR, hopefully https://github.com/open-telemetry/opentelemetry-helm-charts/pull/400

Reply:
Will wait for that PR and then bump the chart version.

Reply:
@joshleecreates if you tested on 0.5.0-alpha, please comment on that PR to confirm it works as expected.

Reply:
> I plan to take care of it soon once things slow down with the demo's feature freeze. 

Agreed, this makes the most sense.

> I'd prefer the image bump to be in its own PR but if things can't go independently I understand. If we do need to bump the version in this PR then the chart's version should bump the minor version instead of the patch. 

I leave it up to your discretion how you'd like to merge things. I've tested all of the variations and here is the status:

- with #400, (as with the current version of the chart), the shipping service fails because the quote service is not present and the shipping service has no address for it.
- with this PR and app release 0.4.0-alpha, the shipping service error is fixed and the communication between shipping/quote service seems to working, but the quote service fails to send traces to the collector.
- with this PR and app release 0.5.0-alpha, all services work and send traces.

> Also, please handle merge conflict.
✅

Reply:
@joshleecreates 0.5.0-alpha change merged in via #400.  Please rebase.